curriculum/challenges/english/01-responsive-web-design/basic-css-cafe-menu/part-077.md

```diff
@@ -8,10 +8,9 @@ isHidden: true
 ## Description
 <section id='description'>
 
-You change properties of a link when the link is actually clicked by using a <dfn>pseudo-selector</dfn> that looks like `a:active { propertyName: propertyValue }`
-
-Change the color of the footer `Visit our website` link to be `white` when a clicks on it.
+Changing the `bottom-margin` to `5px` looks great. However, now the space between the `Cinaommon Roll` menu item and the second `hr` element does not match the space between the top `hr` element and the `Coffees` heading.
```

Comment:
Changing the `bottom-margin` to `5px` looks great. However, now the space between the `Cinnamon Roll` menu item and the second `hr` element does not match the space between the top `hr` element and the `Coffees` heading.